soh/src/code/z_sram.c

```diff
@@ -7,7 +7,6 @@
 #include "soh/Enhancements/randomizer/adult_trade_shuffle.h"
 
 #define NUM_DUNGEONS 8
```

Comment:
I must have missed these, can we remove them all?

Reply:
why would we get rid of those?

Reply:
They aren’t in use anymore as of the random Inf change

Reply:
i don't see how the inf change would make it so we don't want to avoid magic numbers when looping through the number of dungeons/trials etc.

Reply:
Yeah, I guess in the example  of NUM_TRIAL being used in another place it makes sense, but prior to this these macros weren't used for anything other than initializing the arrays in this file, which have since been replaced

Reply:
I didn't remove it though, I needed to access it somewhere else. I'm definitely not in favor of removing them at the moment.